src/jit/compiler.h

```diff
@@ -5710,6 +5711,7 @@ class Compiler
                 __int64 lconVal;
                 double  dconVal;
                 Range   u2;
+                bool    edge;
```

Comment:
Instead on have 'edge' be a bool, how about storing either VNF_GT_UN or VNF_LE_UN as its value.